daphne_worker/src/storage_proxy.rs

```diff
@@ -0,0 +1,136 @@
+use daphne_service_utils::do_requests::{DoRequest, IdFrom, DO_PATH_BASE, KV_PATH_BASE};
+use url::Url;
+use worker::{js_sys::Uint8Array, Env, Request, RequestInit, Response};
+
```

Comment:
Add a docucomment explaining what a "storage proxy" is. Also, please make sure this role is documented in the functional spec (the name should also match).

Related: Does this code share any infrastructure with `daphen_worker` other than `DoRequest`?